releases/latest/beta/Dockerfile.ubi.openjdk17

```diff
@@ -1,12 +1,12 @@
-FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-jdk-ubi
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-ea-jdk-ubi-amd64
```

Comment:
Is this an early driver from Java team with InstantOn capability? Hope multi-arch support would be available for GA

Reply:
To we build multi-arch beta images?

Reply:
> Hope multi-arch support would be available for GA

For GA images the CRIU support will come from the GA semeru images.  For beta images we thought it would be best to continue to use the EA images each beta image release.  Perhaps after GA Semeru images are available we would move our beta images to move back to GA Semeru images.  My position is that for our betas we should test out the latest EA from Semeru to get early access testing of Semeru with our betas.

Reply:
UBI-based Liberty beta image is not multi-arch, but it should be. Something we must address.

Thanks for confirming GA Semeru images will include the CRIU support

Reply:
> UBI-based Liberty beta image is not multi-arch, but it should be. Something we must address.

Semeru EA should have multi-arch support in the future.  It just does not now.  If we need to address this before they provide multi-arch then we can move back to GA semeru images once they are available with the InstantOn (CRIU) support we require.  That will be very soon.